eva-accession-core/src/test/java/uk/ac/ebi/eva/accession/core/summary/DbsnpClusteredVariantSummaryFunctionTest.java

```diff
@@ -0,0 +1,127 @@
+/*
+ *
+ * Copyright 2018 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package uk.ac.ebi.eva.accession.core.summary;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import uk.ac.ebi.eva.accession.core.ClusteredVariant;
+import uk.ac.ebi.eva.accession.core.IClusteredVariant;
+import uk.ac.ebi.eva.accession.core.VariantType;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotEquals;
+
+public class DbsnpClusteredVariantSummaryFunctionTest {
+
+    private static final String ASSEMBLY_ACCESSION = "assembly";
+
+    private static final int TAXONOMY_ACCESSION = 1;
+
+    private static final String CONTIG = "contig";
+
+    private static final int START = 1;
+
+    private static final VariantType VARIANT_TYPE = VariantType.SNV;
+
+    private static final Boolean VALIDATED = null;
+
+    private DbsnpClusteredVariantSummaryFunction summaryFunction;
+
+    @Before
+    public void setUp() {
+        summaryFunction = new DbsnpClusteredVariantSummaryFunction();
+    }
+
+    @Test
+    public void summaryFunctionMustBeIdempotent() {
+        IClusteredVariant ClusteredVariant1 =
```

Comment:
Variable names shouldn't be capitalized. Applies to all the methods in this class.